README.md

```diff
@@ -1,11 +1,13 @@
+# freeCodeCamp README
```

Comment:
```suggestion
```
There's no need, as far as I can see, to have the top header be h1 - the entire readme appears inside an h1 on Github.

Reply:
Sure. My markdown linter was complaining. Should amend it :)

Reply:
Mine too.  It's a good thing for it to highlight, it just doesn't seem to matter in this case.